scripts/variantstore/wdl/GvsPrepareCallset.wdl

```diff
@@ -17,10 +17,12 @@ workflow GvsPrepareCallset {
         String fq_temp_table_dataset = "~{destination_project}.temp_tables"
         String fq_destination_dataset = "~{destination_project}.~{destination_dataset}"
 
+        Int temp_table_ttl_in_hours = 72
+
         String? docker
     }
 
-    String docker_final = select_first([docker, "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_20200414"])
+    String docker_final = select_first([docker, "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_20200507"])
```

Comment:
at some point we should start using 2021 :)

Reply:
lol -- yes, today is the day :smh: